queries/r/locals.scm

```diff
@@ -0,0 +1,11 @@
+; locals.scm
+
+(function_definition) @local.scope
+
+(formal_parameters (identifier) @local.definition)
```

Comment:
we have different captures `@definition.parameter`, `@reference`, @definition.var`